{"uuid": "b45885b5-7f81-4d0a-ab2c-1bde12867580", "children": ["61b1d4b0-707e-40a7-8582-7b38baccf0e1"], "befores": [{"name": "reset_scenario_lf", "status": "passed", "start": 1702295949386, "stop": 1702295959392}], "afters": [{"name": "reset_scenario_lf::1", "status": "passed", "start": 1702296048164, "stop": 1702296048164}, {"name": "reset_scenario_lf::teardown_session", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"/usr/local/lib/python3.8/dist-packages/allure_commons/_allure.py\", line 200, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/home/imgd/ctpriv-scripts/altice/wlan-testing/tests/e2e/basic/conftest.py\", line 152, in teardown_session\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/../libs/lanforge/lf_tools.py\", line 470, in reset_scenario\n    self.Chamber_View()\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/../libs/lanforge/lf_tools.py\", line 503, in Chamber_View\n    self.CreateChamberview.clean_cv_scenario(scenario_name=self.scenario_name)\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/../lanforge/lanforge-scripts/py-scripts/create_chamberview.py\", line 66, in clean_cv_scenario\n    self.rm_cv_text_blob(cv_type, scenario_name)\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/lanforge/lanforge-scripts/py-json/cv_test_manager.py\", line 240, in rm_cv_text_blob\n    self.json_post(req_url, data)\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/lanforge/lanforge-scripts/py-json/LANforge/lfcli_base.py\", line 256, in json_post\n    json_response = lf_r.json_post(show_error=debug_,\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/lanforge/lanforge-scripts/py-json/LANforge/LFRequest.py\", line 176, in json_post\n    resp = urllib.request.urlopen(myrequest)\n  File \"/usr/lib/python3.8/urllib/request.py\", line 222, in urlopen\n    return opener.open(url, data, timeout)\n  File \"/usr/lib/python3.8/urllib/request.py\", line 525, in open\n    response = self._open(req, data)\n  File \"/usr/lib/python3.8/urllib/request.py\", line 542, in _open\n    result = self._call_chain(self.handle_open, protocol, protocol +\n  File \"/usr/lib/python3.8/urllib/request.py\", line 502, in _call_chain\n    result = func(*args)\n  File \"/usr/lib/python3.8/urllib/request.py\", line 1383, in http_open\n    return self.do_open(http.client.HTTPConnection, req)\n  File \"/usr/lib/python3.8/urllib/request.py\", line 1358, in do_open\n    r = h.getresponse()\n  File \"/usr/lib/python3.8/http/client.py\", line 1348, in getresponse\n    response.begin()\n  File \"/usr/lib/python3.8/http/client.py\", line 316, in begin\n    version, status, reason = self._read_status()\n  File \"/usr/lib/python3.8/http/client.py\", line 277, in _read_status\n    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")\n  File \"/usr/lib/python3.8/socket.py\", line 669, in readinto\n    return self._sock.recv_into(b)\n"}, "start": 1702296048164, "stop": 1702296048311}], "start": 1702295949386, "stop": 1702296048312}